### DISCUSSION TITLE
New icons?

### DISCUSSION THREAD

**Discussion by raczkaj** [lucide-icons/lucide#2369] [Ideas]
Hi, is there possible to add some new icons for plane seats? Examples from google: https://fonts.google.com/icons?icon.query=airline+seat

Thanks

**Comment by karsa-mistmere** [comment]
Please [submit an icon request issue](https://github.com/lucide-icons/lucide/issues/new?assignees=&labels=%F0%9F%99%8C+icon+request&projects=&template=01_icon_request.yml), but keep in mind the discussions at #2367
